app/models/rental.rb

```diff
@@ -1,4 +1,7 @@
 class Rental < ApplicationRecord
   belongs_to :user
   belongs_to :camera
+
```

Comment:
Need to add enums for status column: 
https://betterprogramming.pub/how-to-use-enums-in-rails-6-87600e292476